.circleci/config.yml

```diff
@@ -3,7 +3,7 @@ version: 2
 jobs:
     build-and-test:
         macos:
-            xcode: "9.1.0"
+            xcode: "10.0"
```

Comment:
I think you might have to make this `10.0.0` - builds are puking with just `10.0`